docs/syntax/scss.md

```diff
@@ -293,10 +293,22 @@ When listing multiple comma-separated selectors, each one *must* be placed on a
 
 ### Files and folders
 
-* Sass variables, mixins and functions should be in their own files, separate from the code that uses them
+Sass variables, mixins and functions *should* be in their own files, separate from the code that uses them.
 
 
-### Comments
+### Comments and documentation using SassDoc
+
+#### SassDoc
+
+[SassDoc](http://sassdoc.com/) is a documentation generator produce browsable documentation from parsing comments in `*.scss` files.
+
+[View an example of generated documentation](http://sass-mq.github.io/sass-mq/)
```

Comment:
These top two paras could be an `aside`, since they're non normative.  You can also hide asides and have them reveal on clicking a link (see techdocs documentation for examples) which could work well here, eg "What is sassdoc" in brackets after the first mention of SassDoc below.